src/commands/dev/index.js

```diff
@@ -93,35 +86,12 @@ const startFrameworkServer = async function ({ settings, log, exit }) {
       port: settings.frameworkPort,
       output: 'silent',
       timeout: FRAMEWORK_PORT_TIMEOUT,
+      ...(settings.disableLocalServerPolling ? {} : { protocol: 'http' }),
```

Comment:
This option documented via the library types and somehow [here](https://github.com/dwmkerr/wait-port#usage) too.

Reply:
@eduardoboucas can you verify this doesn't break the original fix (I remember you had issues with this when running `create-react-app`).

Reply:
> @eduardoboucas can you verify this doesn't break the original fix (I remember you had issues with this when running `create-react-app`).

It seems to work fine! I remember the issue was with a Gatsby site and I'm not able to replicate it using this branch.